fury/tests/test_text.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import shutil
+from fury import text_tools
+from fury import actor, window
+import fury
+
+
+def test_create_atlas():
+    if not text_tools._FREETYPE_AVAILABLE:
+        print('Bitmap text not tested (FREETYPE is not available)')
+        return
+
+    name = 'test_fonts_1238210930'
+    font_path_atlas = text_tools._FONT_PATH_USER
+    font_path = f'{fury.__path__[0]}/data/files/FreeMono.ttf'
+    text_tools.create_new_font(
+        name=name,
+        font_size_res=10, font_path=font_path, show=False)
+    fonts = text_tools.list_fonts_available(True)
+    shutil.rmtree(f'{font_path_atlas}/{name}')
+    if name not in fonts.keys():
+        raise FileNotFoundError(f'Font {name} was not created')
+
+
+def test_atlas():
+    if not text_tools._FREETYPE_AVAILABLE:
+        print('Bitmap text not tested (FREETYPE is not available)')
+        return
+
+    texture_atlas = text_tools.TextureAtlas(num_chanels=1)
+    font_path = f'{fury.__path__[0]}/data/files/FreeMono.ttf'
+    tf = text_tools.TextureFont(texture_atlas, font_path, 100)
+    tf.load('a sdf123')
```

Comment:
same comment as above ? what is the test here ?

Reply:
If the method responsible to create the atlas is working or not. Yeah, test_atlas is not a good name, I'll improve that

Reply:
I mean, your tests do not have any assertions. so not sure what you are really testing, they look like mini examples. 
The name is ok.

Reply:
Ok, I get your point now. This only verifies that the file is written in the proper location and it's not corrupted. In both cases, the exceptions will be raised. But we can put a try, except and raise to become more meaningful